### ISSUE TITLE
Sync hangs when adding a device

### ISSUE THREAD

**Issue by tomlowenthal** [brave/brave-browser#3428]
<img width="725" alt="screen shot 2019-02-20 at 11 12 24 am" src="https://user-images.githubusercontent.com/639937/53117780-83dbc100-3500-11e9-905b-1bfb9ce13cb8.png">

Tried to set up sync between my MacBook Pro running Version 0.62.4 Chromium: 73.0.3683.39 (Official Build) dev (64-bit) and a fresh Pixel 3 XL running Android 9 Build/PD1A.180720.030 with Brave 1.0.77, Chromium 71.0.3578.99.

After scanning the QR code on the mobile device, the laptop page at <brave://sync> has been showing this for about half an hour.

**Comment by HennyBro** [comment]
I have the EXACT same issue with my first-gen Pixel XL running Android Q BETA 2 with any version of Brave and my new Windows 10 laptop running Windows 10.

**Comment by HennyBro** [comment]
Update: My the Sync WORKS NOW! I installed the latest RC APK for Android and coupled it with the latest stable release from Brave Desktop. It works! It needed about 30 seconds to recognise the device. But it didn't do it by itself. I had to initiate it, I think.

What I did was the following: On my Laptop head over to the sync page and start a new sync chain. Then I chose "pair with new phone/tablet". After that I continued on my phone > sync > adding it to a sync chain > scanned the QR code > waited for a few seconds. I then went ahead and pressed back on my phone, just to see that it (FINALLY) had recognised my Laptop as a sync chain OP.
